src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -68,6 +67,7 @@
 from chip.interaction_model import InteractionModelError, Status
 from chip.setup_payload import SetupPayload
 from chip.storage import PersistentStorage
+from chip.testing.commissioning import CommissioningInfo, CustomCommissioningParameters, SetupPayloadInfo, commission_devices
```

Comment:
There's definitely a few tests that use SetupPlayloadInfo etc. I'm a bit surprised these are still passing without changes to those files. Are these aliased into this file somewhere?

Reply:
I've merged the latest master to my branch and didn't find any examples of usage of `SetupPayloadInfo` in tests.

Reply:
Ah, so they just use the function output without naming it, which might be why this is working OK

https://github.com/project-chip/connectedhomeip/blob/78092a6ba6ccc7bd769dda88cb4d5bbe187d45c1/src/python_testing/TC_SC_7_1.py#L103

Reply:
Resolving then